doa.css

```diff
@@ -0,0 +1,81 @@
+ /*doa.css 1st group project, 2nd iteration - Team Daughters of Anarchy*/
+
+/* GLOBAL FONT */
+@import url(https://fonts.googleapis.com/css?family=Metal+Mania);
+
+* {
+  box-sizing: border-box;
+}
+
+
+/*NAVBAR*/
+
+
+#header {
+  border: 1px solid red;
+  position: fixed;
+  width: 100%;
+
+}
+#header h1 {
+  font-size: 2.5em;
+  font-weight: bold;
+}
+
+
+
+
+.navHome {
+ font-size: 5em;
+ font-weight: bold;
+}
+
+#navLinks {
+font-weight: bold;
+font-size: 3em;
+}
+
+
+
+
+
+
+/*BODY*/
```

Comment:
I think the best flow would be body, .mainContent, #splashPicture, #splashPicture h1